draft-ietf-dnsop-svcb-https.md

```diff
@@ -1004,7 +1008,7 @@ For example, the following is a valid list of SvcParams:
     ech=... key65333=ex1 key65444=ex2 mandatory=key65444,ech
 
 In wire format, the keys are represented by their numeric values in
-network byte order, concatenated in ascending order.
+network byte order, concatenated in strictly increasing numeric order.
```

Comment:
we used the phrase "strictly increasing numeric order" earlier, so this seeks consistency.